frontend/src/features/link-shortener/queries.ts

```diff
@@ -2,10 +2,15 @@ import { useQuery, UseQueryResult } from 'react-query'
 
 import { getGoLinkSuffix } from './GoGovService'
 
+export const goGovKeys = {
+  all: ['gogov'] as const,
+  id: (id: string) => ['gogov', id] as const,
```

Comment:
Was thinking it might be useful in the future 😅 but after discussion with HQ will remove it, whenever we need it can add it back next time

[cf19db9](https://github.com/opengovsg/FormSG/pull/6776/commits/cf19db96721c9a2d195f6f8e15f9e65b6fee536e)